### DISCUSSION TITLE
Issues with rda() "triplot" vectors

### DISCUSSION THREAD

**Discussion by ArthurVDbio** [vegandevs/vegan#659] [Q&A]
### hope this is appropriate ###

CONTEXT
I use the function rda() to analyse morpho-ecological correlations; having  morphological (shapes variables coming from geometric morphometrics) and ecological (10 variables, ordinal or nominal) datasets. 
The code, cca.object seem to be okay and I double checked for the dataset and inputs.

ISSUE
The triplot showes vectors ('bp') with random added letters instead of the true factor level... L and Q are not factor levels and understanble for me.

I hope somebody can help.
Thanks in advance.

DETAILS
My basic functions: 
rda<-rda(coords~.,data=traits.f)
plot(rda,scaling=2,display="bp")

The triplot:
![image](https://github.com/vegandevs/vegan/assets/145691869/faf9a7e2-cbf1-428e-b8c7-f3ba1a02a99e)

**Comment by jarioksa** [comment]
You have *ordered* factors which are expressed as linear (`.L`) and quadratic (`.Q`) contrasts and these effects are shown in graph as arrows. The behaviour is intended and documented. If you define `display="cn"` instead of `"bp"`, you will get the centroids of factor levels *in addition to* these arrows. If you do not want to get arrows for ordered factors, use unordered factors which will only be shown as the centroids of their factor levels.
